aws_cloudtrail_rules/aws_console_root_login.py

```diff
@@ -11,3 +11,19 @@ def title(event):
     return 'AWS root login detected from [{ip}] in account [{account}]'.format(
         ip=event['sourceIPAddress'],
         account=lookup_aws_account_name(event.get('recipientAccountId')))
+
+
+def dedup(event):
+    # Each Root login should generate a unique alert
+    return '-'.join(
+        [event['recipientAccountId'], event['eventName'], event['eventTime']])
+
+
+def alert_context(event):
+    return {
+        'sourceIPAddress': event['sourceIPAddress'],
+        'userIdentityAccountId': event['userIdentity']['accountId'],
+        'userIdentityArn': event['userIdentity']['arn'],
+        'eventTime': event['eventTime'],
+        'mfaUsed': event.get('additionalEventData', {}).get('MFAUsed')
+    }
```

Comment:
This is great!

Reply:
Yeah! We can even make it a helper if we like it